src/patternfly/components/Alert/alert-title.hbs

```diff
@@ -1,6 +1,6 @@
-<h4 class="pf-c-alert__title{{#if alert-title--modifier}} {{alert-title--modifier}}{{/if}}"
+<span class="pf-c-alert__title{{#if alert-title--modifier}} {{alert-title--modifier}}{{/if}}"
```

Comment:
Toast alerts are added to the DOM out of source order from the rest of the content, this is where the added challenge comes in. Does that make sense?

I suppose another way of treating this could be to change where the toast notification (AlertGroup) container is injected within the DOM and instead utilize a region role and assign a sensible accessible name to it. This would have the effect of allowing users to navigate to the entire AlertGroup as a region and from there they could access individual toast notifications. Then, the need for a heading element isn't so great because there are alternate ways to find and consume the content.

In the below screenshot, I changed the AlertGroup's `ul` to live inside of a `section` element instead of `div` and gave the name "Toast notifications" just as an example. I then moved that containing element to be a sibling to `.pf-c-page__main`. Thoughts on this approach? 

<img width="1680" alt="toast-notification-landmark" src="https://user-images.githubusercontent.com/5942899/86616234-89199e80-bf83-11ea-9b48-9f60342c1884.png">